build/yaml/templates/build.yml

```diff
@@ -0,0 +1,29 @@
+#variables:
+#  PackageVersion: Define this in the calling pipeline.
+#  PublishTestResults: (optional) Define this in the calling pipeline.
+
+steps:
+- task: colinsalmcorner.colinsalmcorner-buildtasks.tag-build-task.tagBuildOrRelease@0
+  displayName: tag build
+  inputs:
+    tags: |
+     v: $(PackageVersion)
+  continueOnError: true
+  condition: and(succeeded(), eq(variables['System.PullRequest.IsFork'], 'false'))
+
+- task: NodeTool@0
+  displayName: use node $(NodeVersion)
+  inputs:
+    versionSpec: $(NodeVersion)
+
+# - task: Cache@2
```

Comment:
Should this task be removed?

Reply:
Probably, it is unfortunately super slow. Luckily, yarn install is pretty fast.